api/config/rolepermission.php

```diff
@@ -649,14 +707,23 @@
                 'any' => ['view'],
             ],
             'applicationStatus' => [
-                'any' => ['update'],
+                'any' => ['view', 'update'],
+            ],
+            'applicationNotes' => [
+                'any' => ['view', 'update'],
             ],
             'searchRequest' => [
                 'any' => ['view', 'update', 'delete'],
             ],
             'user' => [
                 'any' => ['view'],
             ],
+            'assessmentPlan' => [
+                'any' => ['view']
+            ],
```

Comment:
Will need to update frontend at some point as currently request responders have no way to use this permission as they cannot get to processes. 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/0f3fec25-8732-4a7e-baf1-2661c838bbcc)

Reply:
Request responders don't need to see the assessment plan directly